.github/workflows/build-and-publish-docs.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - master
```

Comment:
given the branch name change, I guess "master" is not needed?

Reply:
I left `master` in there because the PR is pulling from `petergtz:master`. I can't change the source branch for this PR (would have to close and re-open a new one). But to be able to still see if the GitHub action works in my fork on `master`, I need both. We can immediately remove after merge.